src/fluree/db/graphdb.cljc

```diff
@@ -81,7 +81,7 @@
            idx?-map             (into {} (map (fn [p] [p (dbproto/-p-prop db :idx? p)]) add-preds))
            ref?-map             (into {} (map (fn [p] [p (dbproto/-p-prop db :ref? p)]) add-preds))
            flakes-bytes         (flake/size-bytes add-flakes)
-           schema-change?       (schema-util/schema-change? add-flakes)
+           schema-change?       (schema-util/system-change? add-flakes)
```

Comment:
Does that pred fn still answer the same question? Should we change the name of this var too?